Header header
BboxMsg[] bbox
sensor_msgs/Image image
